packages/@okta/vuepress-site/docs/guides/sign-in-with-linkedin/configure-idp/index.md

```diff
@@ -4,13 +4,13 @@ title: Configure LinkedIn as an Identity Provider in Okta
 
 2.1. Sign in to your Okta org.
 
-2.2. On the main page, hover your cursor over **Users** until the menu opens, then click on **Social & Identity Providers**.
+2.2. If you use the Developer Console, hover your cursor over **Users** until the menu opens, then click on **Social & Identity Providers**. If you use the Classic UI, hover over **Security** then click on **Identity Providers**.
 
 2.3. On the Identity Providers page, click on **Add Identity Provider** > **Add LinkedIn**
 
 * **Name:** We suggest using the name you would expect to see on a button, something like "Log in to LinkedIn".
-* **Client Id:** Paste in the App ID that you got from LinkedIn in step 1.3 above.
-* **Client Secret:** Paste in the App Secret that you got from LinkedIn in step 1.3 above.
+* **Client Id:** Paste in the OAuth client ID that you got from LinkedIn in step 1.3.
+* **Client Secret:** Paste in the OAuth client secret that you got from LinkedIn in step 1.3.
```

Comment:
**Client Secret:** Paste the OAuth client secret that you got from LinkedIn in step 1.3.